module/modmanager/manager_test.go

```diff
@@ -521,8 +519,7 @@ func TestModuleReloading(t *testing.T) {
 		// not modularly managed and commands return error.
 		_, err = h.DoCommand(ctx, map[string]interface{}{"command": "kill_module"})
 		test.That(t, err, test.ShouldNotBeNil)
-		test.That(t, err.Error(), test.ShouldContainSubstring,
-			"error reading from server")
+		test.That(t, err.Error(), test.ShouldContainSubstring, "rpc error")
```

Comment:
Just curious; do you know when an `EOF` happens and when we get `conn is closing`?

Reply:
error reading from server happens when the disconnect is initiated by the server and conn is closing when we initiate close on the client side. there seems to be a race in terms of what happens first in CI, but I can't repro on my local

Reply:
Gotcha that tracks; thanks!